examples/nni_data_augmentation/README.md

```diff
@@ -0,0 +1,35 @@
+## Tuun data augmentation
```

Comment:
Two broad things that might be nice to add to readme:
(1) Just a brief (couple-sentence) overview of what this demo carries out.
(2) Reference/link to basenet implementation we use.

Reply:
Added introduction at the beginning of README